.circleci/config.yml

```
@@ -146,3 +157,48 @@ jobs:
                     --attach build/distributions/git-machete-intellij-plugin-*.zip \
                     --message "$tag"$'\n\n'"$change_notes" \
                     $tag
+
+
+  updateEapBuildNumber:
+    executor: docker_executor
+    steps:
+      - checkout
+      - reconfigure_origin_remote
+      - run:
+          name: Configure github user information
```

Nit:
```suggestion
          name: Configure git user information
```

as this data is not specific to GitHub, it's required by git itself